src/App.jsx

```diff
@@ -14,33 +16,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABET = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, { sortField, isReversed }) {
+  const copiedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortField) {
+    copiedGoods.sort((good1, good2) => {
```

Comment:
it's better to use more clear names instead of `good1` and `good 2`

```suggestion
    copiedGoods.sort((firstGood, secondGood) => {
```